pkg/webhook/nad/validator.go

```diff
@@ -131,6 +131,24 @@ func (v *Validator) checkNadConfig(bridgeConf *utils.NetConf, nad *cniv1.Network
 		return fmt.Errorf("config is empty")
 	}
 
+	//skip bridge config validation for type kube-ovn
+	if bridgeConf.Type == utils.CNITypeKubeOVN {
```

Comment:
nit: maybe it's time to rename the variable to something more general, such as `nadConf`, instead of `bridgeConf`.